### PR TITLE
Add servicing channel for docker images

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Add servicing channel for docker images
* update build to add servicing channel
* copy stable images removing short tags
* update stable files from preview folder
* update CI definition

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [x] Not Applicable
  - **OR**
    - [ ] Update [README.powershell.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershell.md)
    - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by TravisEz13** [comment]
cc @adityapatwardhan

File: release/servicing/nanoserver/getLatestTag.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    $CI
+)
+
+$parent = Join-Path -Path $PSScriptRoot -ChildPath '..'
+$repoRoot = Join-Path -path (Join-Path -Path $parent -ChildPath '..') -ChildPath '..'
+$modulePath = Join-Path -Path $repoRoot -ChildPath 'tools\getDockerTags'
+Import-Module $modulePath
+
+# The versions of nanoserver we care about
+$shortTags = @('1709','1803')
```
Comment (anmenaga) [inline release/servicing/nanoserver/getLatestTag.ps1]: Is this still accurate?

File: release/stable/centos7/docker/Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ ARG fromTag=7
 FROM centos:${fromTag}
 
 ARG VCS_REF="none"
-ARG PS_VERSION=6.0.4
+ARG PS_VERSION=6.1.0
```
Comment (anmenaga) [inline release/stable/centos7/docker/Dockerfile]: some other dockerfiles (e.g. Ubuntu's, servercore) also have 6.0.4 as the default; probably also needs to be updated.

**Comment by anmenaga** [comment]
PR Checklist not filled out.

File: release/servicing/nanoserver/getLatestTag.ps1 (hunk 1 above)
Comment (TravisEz13) [reply release/servicing/nanoserver/getLatestTag.ps1]: yes

File: release/stable/centos7/docker/Dockerfile (hunk 2 above)
Comment (TravisEz13) [reply release/stable/centos7/docker/Dockerfile]: We can update these separately.

**Pull request merged**